torchci/rockset/commons/__sql/original_pr_hud_query.sql

```diff
@@ -1,4 +1,4 @@
-with master_commit as (
+with default_branch_commit as (
```

Comment:
I've now updated the Rockset UI to match this code. Please use 09377767e3178ee4 in the prodVersions JSON for this query.